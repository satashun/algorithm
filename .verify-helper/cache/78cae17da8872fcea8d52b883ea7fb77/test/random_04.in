20
489077036 -562980738 -121085230 766561207 -70823934 730791379 -200362324 -820086734 -733996735 804969659 -294676963 -798753133 257373501 344352999 754084564 -673702894 -978435201 820234746 673612807 802942697
686020155 554387169 648098734 567472164 -408636723 -746674497 399722186 812444837 -745258212 402865597 -316092425 743482317 -549114446 687760668 -625922566 34591289 236854886 -679063463 -398970486 -48379951
290070794 -229318475 451884707 -914967749 -844100331 702965348 308988796 -457677547 -661084540 148916677 647628876 -935786392 994239233 649827920 -250501054 840938891 -636524076 -181690469 352822735 -660966247
395511949 -234225483 247299028 -485151000 903658444 -149290477 256104205 856405188 916343838 -785859959 -469721700 314450636 -346426934 -697072311 -747925368 297396807 -389238169 629601144 429564151 -41582110
-800597144 940879786 -94013788 164984001 -652324722 523106409 490178268 160584551 -141472300 -823626508 -790229934 388895153 671049103 -74450991 -111862099 508513829 -459277989 -491120311 350311093 973557689
-355771244 566057913 840221831 -856227274 -925416860 476007686 998475855 584205722 811398558 -800259030 950225061 376516946 820259792 152998905 -148476326 -47627760 848699774 718756139 469388351 -161901868
768712064 -117618360 -700780813 267801362 216790135 -480167556 -810715660 301233436 610447070 -481706612 855654090 -548991908 -83636673 -620127038 -489393618 -301451925 -410638401 352447081 456854776 -676047466
-94024352 512441765 -722376424 222743349 486841186 -747968582 830602978 -178441768 -956891337 -521527909 -102809196 -857761158 384629509 696531754 -115852418 85518218 146082134 376080878 -170595350 -351401414
789702404 -391408899 -815610669 895871173 -911498047 -787693812 -261107771 531271779 264587721 10708835 730712507 -522034221 905532971 -98197372 -150721550 837468531 -277744564 286504014 -865515322 259673478
-496996182 -663593112 -493179158 -156625957 -757280233 -170654458 473452073 -981683071 206618112 726421902 -283620719 -665692141 -105462745 724239245 335262944 -370806297 -835102699 680848692 859311201 589688357
-909006315 364846092 391844417 885124414 -431213113 -892580659 209650536 828885406 -287693928 -245749878 -510453809 -359740529 729660284 -734008512 53515055 -642121595 754855037 -330555809 505992859 -893114033
-370938412 -590426909 -949618746 760766667 -104977612 -39993685 634586263 -958484539 -832082971 -500200561 584385286 -694376612 636133128 864788844 795973491 504466926 -332565916 -831881229 87398073 -929106475
-939125067 -895392244 -349625286 -312954114 -905259418 607496883 539397975 758054866 -658280480 486250134 -964338479 960886579 -604740213 324403954 281755771 -554283840 106203337 -796801956 78044187 -961918711
58449696 -806157338 -88023895 122354867 178523044 447914511 -350828190 -363360911 -603024033 -158757040 -991961700 -849029019 -45312589 125669945 -217593790 -907096202 -626342595 -319471043 -431280618 393945261
-921162928 -770304076 555189246 85943482 -982323195 -84463857 -677864676 509252774 147871911 872292950 763566278 276907460 990513696 -299745156 556212996 -510866765 616906145 591130692 47960022 -140427653
180131325 713563341 545803561 420820033 974610878 -218582754 225562085 -643718063 -626205442 983321434 -370412577 386947964 -464113515 894498215 -307540255 413345490 -333972194 825493934 -5157470 203211342
508022627 345149734 389443946 573332120 -454476070 92498085 -243671192 -979628949 51317903 924868892 395256193 113425231 -547043724 838305470 796844871 -305151218 208903332 -957928347 91890183 510152297
817140920 -104466448 -61556456 -53695200 -58558254 649285529 -765094708 966085819 823103780 -231561376 -460588316 -171027336 -618094247 -842966756 994510911 -20170649 772681109 -992771483 -394400796 -327298124
-801878136 -890058807 253124379 322907332 54330172 -631670474 331335833 -75058539 819200148 595998050 -646905601 927005319 325762869 130141720 -987954102 954746567 -20291929 -100184155 161791557 783862825
-666778224 -511381508 72492446 15855686 -717949993 997101756 351957446 -721144847 353379107 691659419 -631096389 -560076584 439851849 698123449 331370439 -898708041 687298726 -884965046 102014010 846273252
