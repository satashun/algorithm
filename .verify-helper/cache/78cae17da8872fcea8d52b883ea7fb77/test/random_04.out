-17582466536
16 5 11 9 0 3 6 8 18 7 19 2 1 10 4 12 17 13 14 15
